src/test/java/org/broadinstitute/hellbender/tools/spark/sv/KmerTest.java

```diff
@@ -0,0 +1,123 @@
+package org.broadinstitute.hellbender.tools.spark.sv;
+
+import org.broadinstitute.hellbender.utils.test.BaseTest;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+
+/**
+ * Created by tsharpe on 1/12/16.
+ */
+public class KmerTest extends BaseTest
+{
+    @Test
+    public void testDefaultConstruction()
+    {
+        Assert.assertEquals(new Kmer(10).toString(10),"AAAAAAAAAA");
+        Assert.assertEquals(new Kmer(11).toString(11),"AAAAAAAAAAA");
+    }
+
+    @Test
+    public void testConstructionFromStringsAndBytes()
+    {
+        testConstruction("ACGTACGTACGT");
```

Comment:
typically we data providers for this sort of thing

it would look like this:

```
    @DataProvider(name = "baseStrings")
    public Object[][] getBaseStrings(){
        return new Object[][] {
                {"ACGTACGTACGT"},
                {"ACGTACGTACGTC"},
                {"ACGTACGTACGTCC"},
                {"ACGTACGTACGTCCC"},
                {"ACGTACGTACGTCCCC"}
        };
    }

    @Test(dataProvider = "baseStrings")
    public void testConstructionFromStringsAndBytes(String bases)
    {
        Assert.assertEquals(bases,StringKmerizer.toKmer(bases).toString(bases.length()));
        Assert.assertEquals(bases,BytesKmerizer.toKmer(bases.getBytes()).toString(bases.length()));
    }
```